<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>60</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="mmsi_entry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="mmsi_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Filter MMSI (Pisahkan dengan koma):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_set_filter">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_open_map">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Buka Peta</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_refresh">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1000</width>
      <height>430</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>1000</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_stop_sender">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>8</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop Sender</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start_sender">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>8</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start Sender</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop_receiver">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>8</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop Receiver</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start_receiver">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>8</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start Receiver</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
